packages/deprecation-crawler/src/lib/tasks/checkout.ts

```diff
@@ -6,15 +6,13 @@ import {
   getRemoteUrl,
   tap,
 } from '../utils';
-import { ensureGitTag } from './ensure-git-tag';
 
 /**
  * Checkout the desired branch
  * Adds the branch date to the release
  */
-export function checkout(config: CrawlConfig): CrawlerProcess {
+export function checkout(_config: CrawlConfig): CrawlerProcess {
   return concat([
-    ensureGitTag(config),
```

Comment:
Do we still want `ensureGitTag ` here?
Without it, we just run the default command on the current branch.
If we want to keep `ensureGitTag`, I will add it back.
I removed it when I started with the history command because it was causing issues (being asked to checkout two times), but I think that was because of me...